Library/Formula/supervisor.rb

```diff
@@ -0,0 +1,108 @@
+class Supervisor < Formula
+  desc "Process Control System"
+  homepage "http://supervisord.org/"
+  url "https://github.com/Supervisor/supervisor/archive/3.2.1.tar.gz"
+  sha256 "d9c0b17ec42ac6477e7ef0ad2e1f6f1597f855a4d9606c024ac40eb034d6e9ed"
+
+  resource "meld3" do
+    url "https://pypi.python.org/packages/source/m/meld3/meld3-1.0.2.tar.gz"
+    sha256 "f7b754a0fde7a4429b2ebe49409db240b5699385a572501bb0d5627d299f9558"
+  end
+
+  def install
+    # Before the example Supervisor configuration file gets installed, modify
+    # it to use Homebrew's conventions.
+    inreplace buildpath/"supervisor/skel/sample.conf" do |s|
+      # A `[unix_http_server]` modification
+      s.gsub! %r{(?<=^file=)/tmp/supervisor.sock}, var/"run/supervisor.sock"
```

Comment:
@DomT4 If the directories already exist I'm seeing a warning. Should I add something like `unless the_path.exist?`to the `mkdir` lines?